schema/deploy/mutations/create_project_revision_with_revision_type.sql

```diff
@@ -0,0 +1,111 @@
+-- Deploy cif:mutations/create_project_revision_with_revision_type to pg
+
+begin;
+create or replace function cif.create_project_revision_with_revision_type(project_id integer, revision_type varchar(100))
```

Comment:
this could be done by just changing the existing `create_project_revision.sql`  mutation to have a parameter of type `cif.revision_type` that defaults to `null` - this way we can avoid duplicating a lot of code